### PR TITLE
Cronjob for Rasa install and init.

### PR DESCRIPTION
**Pull request by koaning**
As mentioned on [slack](https://rasa-hq.slack.com/archives/C36SS4N8M/p1642506111085900?thread_ts=1642494311.084600&cid=C36SS4N8M), this is an attempt to take [the install cron experiment](https://github.com/RasaHQ/rasa-install-cron).

### CODE REVIEW

**Comment by tczekajlo** [comment]
How is it going to notify a responsible team that something fails?

**Comment by tczekajlo** [comment]
> The same thread is discussing what slack channel to notify. I should say though, I'm unfamiliar with the slack configuration. @tczekajlo when I look at [this workflow](https://github.com/RasaHQ/rasa/blob/main/.github/workflows/ci-model-regression-on-schedule.yml) it seems like I need to add:
> 
> ```yaml
>       - name: Notify slack on failure
>         if: failure()
>         env:
>           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
>         uses: voxmedia/github-action-slack-notify-build@212e9f7a9ca33368c8dd879d6053972128258985
>         with:
>           channel_id: ${{ secrets.SLACK_ALERTS_CHANNEL_ID }}
>           status: FAILED
>           color: danger
> ```
> 
> Would this suffice?

Yep, it should do the job.

**Comment by koaning** [comment]
@tczekajlo just added the snippet 👍

File: .github/workflows/rasa-install-cron-check.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# This is a installation test that we run daily. We attempt to install Rasa
+# on Ubuntu, Mac and Windows and try to run `rasa init`. The goal here is to
+# catch installation issues in an automated way.
+
+name: Cron Test
```
Comment (tczekajlo) [inline .github/workflows/rasa-install-cron-check.yml:5]: I'd use a more descriptive name, something like "pip installation test".

**Comment by tczekajlo** [comment]
> @tczekajlo I figured with this PR I can also go ahead and delete the original repository. I've been able to archive it, but it seems I'm not allowed to delete it. Are you able to remove [this repo](https://github.com/RasaHQ/rasa-install-cron) or should I check with Tom?

I'll take care of it.

**Review by tczekajlo — APPROVED** [review]
LGTM

**Pull request merged**